features/FEATURE_BLE/targets/TARGET_CORDIO/doc/PortingGuide.md

```diff
@@ -1,91 +1,78 @@
-# Porting guide 
+## Porting guide
```

Comment:
Why heading starts at level 2 ?

Reply:
Level 1 breaks our JSON engine.

Reply:
Irit responded below- the tags don't render in our engine, sorry.